tests/com/goide/completion/GoCompletionTest.java

```diff
@@ -437,7 +437,7 @@ public void testPackageNames() {
     myFixture.configureByText("test_file.go", "package fromFile");
     myFixture.configureByText("test.go", "package <caret>");
     myFixture.completeBasic();
-    assertSameElements(myFixture.getLookupElementStrings(), "fromTest", "fromTest_test", "fromFile", "main");
+    assertSameElements(myFixture.getLookupElementStrings(), "fromTest", "fromFile", "main");
```

Comment:
1. completion still should suggest *_test packages in test files. Add test for that.
2. does following code in GoCompletionContributor is still needed?
```
if (packageName.endsWith(GoConstants.TEST_SUFFIX)) {
          result.addElement(LookupElementBuilder.create(StringUtil.trimEnd(packageName, GoConstants.TEST_SUFFIX)));
        }
```